scripts/run_kops_test.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# This script creates kOps cluster and runs full integration tests
+set -euoE pipefail
+
+cleanup(){
+
+  exitcode=$?
+  if [[ ${exitcode} -gt 0 ]]; then
+    printf "Error occurred. Attempting to clean setup.. \n"
+  else
+    printf "Tests finished. Attempting to clean setup.. \n"
+  fi
+
+  if ! uninstall_aws_lb_controller; then
+    printf "Uninstalling AWS LB Controller failed.. \n"
+  fi
+
+  if [[ ! -z "${policy_arn}" ]]; then
+    if ! detach_policy_to_role "${policy_arn}" "nodes.${CLUSTER_NAME}"; then
+      printf "Detaching policy from role failed.. \n"
+    fi
+
+    if ! iam::delete_policy "${policy_arn}" "${REGION}"; then
+      printf "Delete policy failed.. \n"
+    fi
+  fi
+
+  if ! delete_kops_cluster; then
+    printf "Delete cluster failed.. \n"
+  fi
+}
+
+: "${TEST_ID:=$RANDOM}"
+: "${REGION:='us-west-2'}"
```

Comment:
You could use :- to assign default value value only if it is missing. This provides a useful way to override the defaults when you invoke the scripts. For example,
REGION=${REGION:-"us-west-2"}